### PR TITLE
[plugin.video.sandmann] v1.1.0

### PR DESCRIPTION
**Pull request by sorax**
### Description
- **add**: reload functionality (can be configured in the addon-settings)
- **fix**: images not loading

### Checklist:
- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

File: plugin.video.sandmann/addon.py [hunk 1]
```diff
@@ -16,25 +16,53 @@
...
+quality = addon.getSettingInt("quality")
+update = addon.getSettingInt("update")
+interval = addon.getSettingInt("interval")
+
+title, description, thumbnail_image, fanart_image, stream = getEpisodeData(
     episodes_url, quality)
 
-li = xbmcgui.ListItem(label=title, thumbnailImage=thumbnail_image)
-li.setProperty("fanart_image", fanart_image)
+li_episode = xbmcgui.ListItem(label=title, thumbnailImage=thumbnail_image)
```
Comment (enen92) [inline plugin.video.sandmann/addon.py]: Please remove thumbnailImage kwarg and use the setArt method on the ListItem object:

```python
li_episode.setArt({'thumb':thumbnail_image, 'fanart':fanart_image})
```

File: plugin.video.sandmann/addon.py [hunk 2]
```diff
@@ -16,25 +16,53 @@
...
+li_episode.setProperty("fanart_image", fanart_image)
```
Comment (enen92) [inline plugin.video.sandmann/addon.py]: Drop please

File: plugin.video.sandmann/addon.py (hunk 1 above)
Comment (sorax) [reply plugin.video.sandmann/addon.py]: okay, I'll do this in the next version.

File: plugin.video.sandmann/addon.py (hunk 2 above)
Comment (sorax) [reply plugin.video.sandmann/addon.py]: okay

**Pull request merged**